### ISSUE TITLE
Remove deprecated `RowMetadata.getColumnNames()` method

### ISSUE THREAD

**Issue by mp911de** [r2dbc/r2dbc-mssql#247]

**Comment by mp911de** [comment]
Fixed via 8696e12.

**Comment by mofeed-hassan** [comment]
@mp911de Hello Mark, I am new to R2DBC and I faced while developing the problem mentioned here (depricated method problem). I am using r2dbc vesion Borca-SR1.
May I ask if this problem is already resolved or not? If yes which version of r2dbc should I use? I appreciate your help so much.

**Comment by mp911de** [comment]
There was no problem in the first place. This ticket is about removing a deprecated method for version 1.0.

**Comment by mofeed-hassan** [comment]
Thanks for the fast reply. The point is that I am facing the exception "**Failed to obtain R2DBC, this publisher does not support multiple subscribers**". During search I understood that the cause is the method connectionPool.create() returns a Mono connection that has already subscribed to.  I used the r2dbc-bom version [Borca-SR1] as it should contain the Oracle r2dbc fix for it, but it raises the exception of "This method is deprecated for removal". As Understand this method is now removed, so I am expecting r2dbc version that has solution for the original exception without the side effects of depricating RowMetadata.getColumnNames(), and that was my question is there such a version expected?
Sorry for long description, but I wanted to clarify as much as possible. Thanks for help.
